logging/config_test.go

```diff
@@ -131,7 +131,7 @@ func TestNewConfigNoEntry(t *testing.T) {
 
 func TestNewConfig(t *testing.T) {
 	wantCfg := "{\"level\": \"error\",\n\"outputPaths\": [\"stdout\"],\n\"errorOutputPaths\": [\"stderr\"],\n\"encoding\": \"json\"}"
-	wantLevel := zapcore.InfoLevel
+	wantLevel := zapcore.ErrorLevel
```

Comment:
any reason for this change?

Reply:
I want to show that the ConfigMap was actually parsed correctly. InfoLevel is the zero value, so will be returned if that key is not set for some reason.